File: src/com/pokosho/PokoshoException.java
package com.pokosho;

import java.io.IOException;

public class PokoshoException extends Exception {
	/**
	 * serialVersionUID.
	 */
	private static final long serialVersionUID = -2105145060837356952L;

	public PokoshoException() {
		super();
	}

	public PokoshoException(IOException e) {
		super(e);
	}

	public PokoshoException(String s) {
		super(s);
	}

	public PokoshoException(Throwable e) {
		super(e);
	}
}

File: src/main/kotlin/com/pokosho/PokoshoException.kt
package com.pokosho

import java.io.IOException

class PokoshoException : Exception {

    constructor() : super() {}

    constructor(e: IOException) : super(e) {}

    constructor(s: String) : super(s) {}

    constructor(e: Throwable) : super(e) {}

    companion object {
        /**
         * serialVersionUID.
         */
        private val serialVersionUID = -2105145060837356952L
    }
}

